src/main/java/seedu/address/model/datetime/DatetimeRange.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.model.datetime;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+
+/**
+ * Represents a start and end datetime in the ModQuik.
+ * Guarantees: immutable
+ */
+
+public class DatetimeRange {
```

Comment:
DatetimeRange seems to suggest 2 inputs of date and 2 inputs of time respectively, foe eg
2022-01-01 08:00 to 2022-03-03 13:00. Should clarify it in the comments

Reply:
I see. Will make changes after Consultation also takes in a date (implicitly refers to both start and end dates)

Reply:
Since each consultation usually lasts for a few hours on a single day, maybe each consultation can take in 1 date (2022-03-03) and 1 time slot (08:00-13:00).

Reply:
So I think we can abstract things out in the following way:
- Class `Timeslot` (i.e. 08:00-13:00) for `Tutorial` & `Consultation` & `Reminder` (if possible)
- Class `Day` (i.e. Monday) for `Tutorial`
- Class `Date` (i.e. 2022-03-03) for `Consultation` & `Reminder`